### ISSUE TITLE
HYCOM - define variable fails

### ISSUE THREAD

**Issue by karlmsmith** [NOAA-PMEL/LAS#599]
**Reported by jing.y.li on 17 Oct 2008 21:33 UTC**
Ansley,

Use your test server, I got error for 'define variable'.[[BR]]

- choose hycom_glb_analysis[[BR]]

- select 'sea water salinity'[[BR]]

- select 'define variable'[[BR]]

- select average on T axis[[BR]]

- select time range of 01-Nov-2003 to 10-Nov-2003[[BR]]

```
http://porter.pmel.noaa.gov:8380/las/ProductServer.do?xml=%3C%3Fxml+version%3D%221.0%22%3F%3E%3ClasRequest+package%3D%22%22+href%3D%22file%3Alas.xml%22+%3E%3Clink+match%3D%22%2Flasdata%2Foperations%2Foperation[%40ID%3D%27Plot_2D_XY%27]%22+%2F%3E%3Cproperties+%3E%3Cferret+%3E%3Cview+%3Exy%3C%2Fview%3E%3Cformat+%3Eshade%3C%2Fformat%3E%3Ccontour_levels+%3E%3C%2Fcontour_levels%3E%3Ccontour_style+%3Edefault%3C%2Fcontour_style%3E%3Cdeg_min_sec+%3Edefault%3C%2Fdeg_min_sec%3E%3Cexpression+%3E%3C%2Fexpression%3E%3Cfill_levels+%3E%3C%2Ffill_levels%3E%3Cimage_format+%3Edefault%3C%2Fimage_format%3E%3Cinterpolate_data+%3Efalse%3C%2Finterpolate_data%3E%3Cland_type+%3Edefault%3C%2Fland_type%3E%3Cmargins+%3Edefault%3C%2Fmargins%3E%3Cmark_grid+%3Eno%3C%2Fmark_grid%3E%3Cpalette+%3Edefault%3C%2Fpalette%3E%3Cset_aspect+%3Edefault%3C%2Fset_aspect%3E%3Csize+%3E0.5%3C%2Fsize%3E%3Cuse_graticules+%3Edefault%3C%2Fuse_graticules%3E%3Cuse_ref_map+%3Edefault%3C%2Fuse_ref_map%3E%3C%2Fferret%3E%3C%2Fproperties%3E%3Cargs+%3E%3Clink+match%3D%22%2Flasdata%2Fdatasets%2Fhycom_glb_analysis%2Fvariables%2Fsalinity%22+%3E%3Canalysis+label%3D%22sea_water_salinity_1%22+%3E%3Caxis+hi%3D%2210-Nov-2003%22+type%3D%22t%22+lo%3D%2201-Nov-2003%22+op%3D%22ave%22+%2F%3E%3C%2Fanalysis%3E%3C%2Flink%3E%3Cregion+%3E%3Crange+low%3D%22-180.0%22+type%3D%22x%22+high%3D%22180.0%22+%2F%3E%3Crange+low%3D%22-78.0%22+type%3D%22y%22+high%3D%2289.92%22+%2F%3E%3Cpoint+v%3D%220%22+type%3D%22z%22+%2F%3E%3C%2Fregion%3E%3C%2Fargs%3E%3C%2FlasRequest%3E&JSESSIONID=157B8534E46A180E539DFF9148E31ADA
```
 
I also get the same error on my updated HYCOM server.

Migrated-From: http://dunkel.pmel.noaa.gov/trac/las/ticket/593

**Comment by karlmsmith** [comment]
**Comment by jing.y.li on 17 Oct 2008 21:39 UTC**
Changed the component to 'Define variable'.

**Comment by karlmsmith** [comment]
**Comment by @AnsleyManke on 17 Oct 2008 22:12 UTC**
This seems to be at least in part due to a timeout, in creating the intermediate fiel F-TDS data.  Small regions work but larger ones do not. Steve and I were both getting correct plots all morning.  Still checking into this.

**Comment by karlmsmith** [comment]
**Comment by jing.y.li on 23 Oct 2008 20:39 UTC**
I increased the time_limit from 200 to 600. It can average over a short time period, for example 11 days, 01-Nov-2003 to 11-Nov-2003.


But when the time range is long, 01-Nov-2003 to 01-Nov-2004, it fails and gets the unknown error code 2002 again.

```
 Cached data cleared from memory
 *** Unknown error code:      2002 ***
 
```

**Comment by karlmsmith** [comment]
**Comment by @AnsleyManke on 23 Oct 2008 23:07 UTC**
Fixed. See #605

**Comment by karlmsmith** [comment]
**Comment by @AnsleyManke on 23 Oct 2008 23:15 UTC**
Fixed; see report under #605

**Comment by karlmsmith** [comment]
**Modified by @AnsleyManke on 23 Oct 2008 23:15 UTC**
